Ecosystem Partners/Partner Integration Resources/ecosystem-program-resources-addons.md

```diff
@@ -1,18 +1,22 @@
 {{{
   "title": "Ecosystem Program Partner Resource Guide - Addons",
-  "date": "3-16-2015",
-  "author": "Keith Resar",
+  "date": "7-15-2015",
+  "author": "Andrew Brunette",
   "attachments": [],
   "contentIsHTML": false
 }}}
 
 
+CenturyLink [Ecosystem Program Partner Members](centurylink-cloud-ecosystem-program-guide.md) gain access to a marketplace of [resources](ecosystem-program-resources.md) that make integrations easy.
 
-CenturyLink [Ecosystem Program Partner Members](centurylink-cloud-ecosystem-program-guide.md) gain access to a bevy of [resources](ecosystem-program-resources.md) that make integrations easy.
+The CenturyLink Cloud AddOn engine allows partners to add their services to the AppFog marketplace, giving customers access to service broker delivered services.  Partners register their service with the AppFog AddOn engine.  The service is then visible to the AppFog user through the AppFog marketplace.  The customer can sign up for the service directly through AppFog.  
 
-The CenturyLink Cloud Add-On engine is coming soon, giving customers access to service broker delivered services.  More to come in this space was the service nears general availability.
+AppFog V2 is currently in beta.  Prospective partners with questions should contact ecosystem@ctl.io
 
 ### Core Documentation
 
-*The Addons service is coming soon.  Documentation will be available once released.*
+  * [Become an AddOn partner](https://www.centurylinkcloud.com/knowledge-base/appfog/legacy-version-1/become-an-add-on-partner/)
```

Comment:
Do not use full qualified links to other KBs. Please Read the README https://github.com/CenturyLinkCloud/PublicKB#links-kb-article-to-kb-article